app/assets/javascripts/controllers/middleware_topology/middleware_topology_controller.js

```diff
@@ -141,18 +141,22 @@ function MiddlewareTopologyCtrl($scope, $http, $interval, $location, topologySer
       window.location.assign(topologyService.geturl(d));
     };
 
+
     self.getCircleDimensions = function getCircleDimensions(d) {
+        var defaultDimensions = topologyService.defaultElementDimensions();
         switch (d.item.kind) {
             case 'MiddlewareManager':
                 return { x: -20, y: -20, height: 40, width: 40, r: 28 };
             case 'Container' :
                 return { x: -7, y: -7,height: 14, width: 14, r: 13 };
             case 'MiddlewareServer' :
                 return { x: -12, y: -12, height: 23, width: 23, r: 19 };
+            case "Vm":
+                return { x: defaultDimensions.x, y: defaultDimensions.y, r: 21 };
             default :
                 return { x: -9, y: -9, height: 18, width: 18, r: 17 };
         }
```

Comment:
that's part of my concerns. the dimensions part changed because the code now mixes dimensions of font stuff and png. and so it is less suitable as it is now. (and the controller sets some dimensions [hard coded](https://github.com/ManageIQ/manageiq/blob/master/app/assets/javascripts/controllers/middleware_topology/middleware_topology_controller.js#L116))